meta/src/title.rs

```diff
@@ -36,11 +36,11 @@ impl std::fmt::Debug for TitleContext {
 
 /// A function that is applied to the text value before setting `document.title`.
 #[repr(transparent)]
-pub struct Formatter(Box<dyn Fn(String) -> String>);
+pub struct Formatter(Box<dyn Fn(Oco<'static, str>) -> Oco<'static, str>>);
```

Comment:
(See comment above — I'd rather keep this one `Fn(String) -> String`, as it ~never needs to be cloned and is additional work for the user)